packages/appstore/src/components/wallets-banner/wallets-banner-upgrade.tsx

```diff
@@ -27,7 +27,9 @@ const WalletsBannerUpgrade = observer(() => {
                     text={localize('Upgrade now')}
                     primary
                     large
-                    onClick={() => toggleWalletsUpgrade(true)}
+                    onClick={() => {
+                        // toggleWalletsUpgrade(true)
```

Comment:
Could you add TODO comment here and the other places where there are commented out codes.